src/view/ViewCommandHandlers.js

```diff
@@ -266,6 +266,23 @@ define(function (require, exports, module) {
         }
     }
 
+
+    /**
+     * Font smoothing setter to set the anti-aliasing type for the code area on Mac.
+     * @param {string} aaType The antialiasing type to be set.It can take either "subpixel-antialiased" or "antialiased"
+     */
+    function setMacFontSmoothingType(aaType) {
+        var $editor_holder  = $("#editor-holder");
+
+        // Add/Remove the class based on the preference. Also
+        // default to subpixel AA in case of invalid entries.
+        if (aaType === "antialiased") {
+            $editor_holder.removeClass("subpixel-aa");
+        } else if (aaType === "subpixel-antialiased" || aaType !== "antialiased") {
```

Comment:
This does not seem right. It does not make sense to have literal `subpixel-antialiased` if its value does not matter. If the value of the setting is not recognized, no action should be taken and a warning should be logged.

Perhaps the entire construct would be better if organized as a `switch` statement.